src/runners/scilla_runner.ml

```diff
@@ -65,7 +64,7 @@ let check_extract_cstate name res gas_limit =
 (*   Running the simularion and printing results     *)
 (*****************************************************)
 
-let check_after_step name res gas_limit  =
+let check_after_step _name res gas_limit  =
```

Comment:
Any suggestions on what to do with `name` parameter are very welcome!

Reply:
I think the parameter can be removed. It doesn't seem useful as the location is already captured in `scilla_error`.

Reply:
Done.